app/views/subtopics/_subtopic.html.erb

```diff
@@ -11,8 +11,8 @@
 
     <div class="govuk-grid-column-one-third add-title-margin">
       <%= render "govuk_publishing_components/components/subscription-links", {
-        email_signup_link: email_signup_path(topic_slug: params[:topic_slug], subtopic_slug: params[:subtopic_slug]),
-        email_signup_link_text: "Subscribe to email alerts"
+        email_signup_link: "/email-signup?topic=/topic/#{params[:topic_slug]}/#{params[:subtopic_slug]}",
```

Comment:
Will we remove/redirect the routes in a separate follow-up PR? https://github.com/alphagov/collections/blob/master/config/routes.rb#L35

Reply:
We could do that in this PR, and add a comment to say they are redundant. [[Example](https://github.com/alphagov/email-alert-frontend/blob/master/config/routes.rb#L40)]